generators/environment/templates/bin/pre-install.sh

```diff
@@ -10,11 +10,6 @@
 # of `grunt install`.
 ##
 
-if [ ! -f src/sites/default/default.settings.php ]; then
-  echo "Please run pre-install after successfully building the docroot."
-  exit 1
-fi
```

Comment:
If default.settings.php does not exist, that means there's no Drupal codebase. If there's no drupal codebase, why would you want the install process to be able to proceed and why would there be stuff in the caching services to wipe?